src/settings.js

```diff
@@ -1,15 +1,26 @@
 require('dotenv').config()
 
-var scheme = process.env.SCHEME || 'http://';
-var host = process.env.HOST || 'localhost';
-var port = process.env.PORT || 4000;
-var graphiql_enabled = process.env.GRAPHIQL_ENABLED || true;
-var graphiql_endpoint = process.env.GRAPHIQL_ENDPOINT || '/graphiql';
+var express = {
+  port: process.env.PORT || 4000,
+  url: (process.env.SCHEME || 'http://') + (process.env.HOST || 'localhost') + ':' + (process.env.PORT || 4000)
+};
+
+var graphiql = {
+  enabled: process.env.GRAPHIQL_ENABLED || true,
```

Comment:
You can never turn this off because `false || true` will always evaluate to `true`